downstyler.css

```diff
@@ -79,6 +79,7 @@ a[href]:visited { color: BlueViolet; }
 
 /* Styles for code-related elements */
 pre, :not(pre) code, kbd, samp, tt { padding: .1em .3em; margin: 0 .1em; font-size: 80%; background: #eee; }
+:is(i, em) code, kbd, samp, tt { transform: skew(-10deg); display: inline-block; font-style: normal; }
```

Comment:
Maybe also include in the `:is()` other tags that are rendered as italics (`<dfn>`, `<var>`, `<cite>`)?